.circleci/config.yml

```diff
@@ -1,248 +1,131 @@
+      #       - composer-v1-
+
+      # - restore_cache: # special step to restore the dependency cache if `package-lock.json` does not change
+      #     keys:
+      #       - node-v1-{{ checksum "package-lock.json" }}
+      #       # fallback to using the latest cache if no exact match is found (See https://circleci.com/docs/2.0/caching/)
+      #       - node-v1-
+
+      - run:  #Invoke all command-line programs
+          name: Install Composer  #Title of the step to be shown in the CircleCI UI
+          command: |
+            php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+            php -r "if (hash_file('SHA384', 'composer-setup.php') === trim(file_get_contents('https://composer.github.io/installer.sig'))) { echo 'Installer verified'; } else { echo 'Installer invalid'; unlink('composer-setup.php'); } echo PHP_EOL;"
+            php composer-setup.php
+            php -r "unlink('composer-setup.php');"
+
+      - run:  #Invoke all command-line programs
+          name: Display PHP information #Title of the step to be shown in the CircleCI UI
+          command: |
+            php -v
+            php composer.phar --version
+            
+      - run:  #Invoke all command-line programs
+          name: Install project dependencies  #Title of the step to be shown in the CircleCI UI
+          command: php composer.phar install -n
+
+      # - run:  #Invoke all command-line programs
+      #     name: Build and Validate  #Title of the step to be shown in the CircleCI UI
+      #     command: |
+      #       export PROJECT_ROOT="$PWD"
+      #       source ~/.profile
+      #       nvm install
+      #       nvm use
+      #       .docksal/commands/npmrc
+      #       .docksal/commands/make
+      #       .docksal/commands/ci-validate
+
+      # - save_cache: # special step to save the dependency cache with the `composer.lock` cache key template
+      #     key: composer-v1-{{ checksum "composer.lock" }}
+      #     paths:
+      #       - vendor
+
+      # - save_cache: # special step to save the dependency cache with the `package-lock.json` cache key template
+      #     key: node-v1-{{ checksum "package-lock.json" }}
+      #     paths:
+      #       - node_modules
+
+      - persist_to_workspace: #Special step used to persist a temporary file to be used by another job in the workflow
+          root: . #Either an absolute path or a path relative to working_directory
+          paths:  #Glob identifying file(s), or a non-glob path to a directory to add to the shared workspace. Interpreted as relative to the workspace root. Must not be the workspace root itself.
+            - web #(working_directory)/var/www/web
+            - vendor  #(working_directory)/var/www/vendor
+            - web/themes/custom/lyndon/lyndon_build #(working_directory)/var/www/web/themes/custom/lyndon/lyndon_build
```

Comment:
Why are you attaching Lyndon? Your project does not use Lyndon. Line 129 should be removed.